protos/telemetry/telemetry.proto

```diff
@@ -85,6 +87,8 @@ service TelemetryService {
     rpc SetRateInAir(SetRateInAirRequest) returns(SetRateInAirResponse) {}
     // Set rate to landed state updates
     rpc SetRateLandedState(SetRateLandedStateRequest) returns(SetRateLandedStateResponse) {}
+    // Set rate to VTOL state updates
+    rpc SetRateVtolState(SetRateVtolStateRequest) returns(SetRateVtolStateResponse) {}
```

Comment:
I would actually not expose that because it's tied to the status anyway, and generally I'd say 1 Hz is good enough.

Reply:
I didn't think much about it, I did more or less a simply copy of the other information extracted from the `EXTENDED_SYS_STATE`

Reply:
Yer makes sense.